src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml.cs

```diff
@@ -20,13 +20,13 @@ namespace APIViewWeb.Pages.Assemblies
     public class IndexPageModel : PageModel
     {
         private readonly IReviewManager _manager;
-        private readonly IHubContext<NotificationHub> _notificationHubContext;
```

Comment:
Are this needed. Does not look like you use them.

Reply:
I think we can keep this in, since we use them on line 88 for notifications. It might've flagged it as added because I changed the name from `NotificationHub` to `SignalRHub`.
```c#
await _notificationHubContext.Clients.Group(User.Identity.Name).SendAsync("RecieveNotification", notifcation);
```

Reply:
Should I change `_notificationHubContext` to `_signalRHubContext`? We only use it for notification hub in this file but it would be more consistent to keep all the names signalRHub-related

Reply:
Ok, that's fine. Perharps change the name to signalRHub